tf2_gnn/data/ppi_dataset.py

```diff
@@ -100,49 +101,27 @@ def __load_data(self, data_dir: RichPath, data_fold: DataFold) -> List[PPIGraphS
         node_to_graph_id = data_dir.join("%s_graph_id.npy" % data_name).read_by_file_suffix()
 
         # We read in all the data in two steps:
-        #  (1) Read features, labels and insert self-loop edges (edge type 0).
-        #      Implicitly, this gives us the number of nodes per graph.
+        #  (1) Read features and labels. Implicitly, this gives us the number of nodes per graph.
         #  (2) Read all edges, and shift them so that each graph starts with node 0.
-        fwd_edge_type = 0
-        if self._params["add_self_loop_edges"]:
-            self_loop_edge_type = 1
-        if not self._params["tie_fwd_bkwd_edges"]:
-            bkwd_edge_type = self.num_edge_types - 1  # Always last
 
-        graph_id_to_graph_data: Dict[int, PPIGraphSample] = {}
+        graph_id_to_edges: Dict[int, List] = {}
```

Comment:
Doing deeper typing here (and similarly in the following lines) with `Dict[int, List[Tuple[int, int]]]` would be nice.